### DISCUSSION TITLE
A fresh start for the season 🧽

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#121534] [Announcements]
For those in the Northern Hemisphere, spring has arrived and with it the notion of spring cleaning. We thought that no matter where you are in the world, we could all use a nudge for tidying up around GitHub and the community.

 

### What We’re Doing :octocat: 
We’ve heard you on the amount of low value and miscategorized posts in the community so we’re working hard behind the scenes as well as in the community to tackle this 🤝. 

You may have seen us taking action against content that violates our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) and making sure posts are in the proper category. As we continue our efforts, we’ll also be encouraging our members to help us respond to posts along with some general cleaning up 🧹. 

### How You Can Help ❣️

Keeping our community tidy is no small feat with our thousands of daily visitors and we’d love your help. Here is how you join in our efforts: 

* **For community members who post questions**: [Marking responses that solves your question](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer) as the answer goes a long way ✅. 
When you mark a comment in the Discussion as an answer, GitHub will highlight the comment and replies to the comment to help other community members quickly find the answer as well.
* **Answer the unanswered**: Share your expertise and help solve the [unsolved](https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+is%3Aunanswered). Everyone here has a unique skill set and yours may be the one that helps unblock a fellow community member 💡.
* **Lead by example**: Be a great community citizen! Ensure you’re posting in the proper category, be courteous and kind to others, and help our [New Members](https://github.com/orgs/community/discussions/categories/new-to-github?discussions_q=is%3Aopen+is%3Aunanswered+category%3A%22New+to+GitHub%22). Showcase what makes the GitHub community so special by being your best self. 
* **Report suspicious content**: See something that is off topic or violates our Code of Conduct? Report it so our team can review. 

### Keep on Cleaning 🧼

These efforts aren’t just limited to the community, take a moment to spruce up the rest of your GitHub presence as well. Some things the Community team will be doing: 
* [Archiving old items in our Project boards](https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-items-in-your-project/archiving-items-from-your-project)
* [Closing](https://docs.github.com/en/issues/tracking-your-work-with-issues/closing-an-issue) and updating stale Issues 
* [Deleting unused forked repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository) 
* Reviewing our profiles and ensuring all information is accurate and up to date 

We hope to have you join us! If you have any tips on how you keep your projects and profile in order - share them below 💗.

**Comment by AL3A31LCS** [comment]
AL3A31

**Comment by jh6143674** [comment]
signal detection and finding probability of error with digital modulation in 5G. We can determine the research paper on this project and code

**Reply by Cinnamonrou** [reply]
[ {
  'event': Test.Event,
  'args': {'a': 69, 'b': '0x12345678901234567890123456789012'},
  'number': n
  },
  { ...
  } ...
]

**Comment by TheMikeyRoss** [comment]
Also the ability to group repositories would be amazing! People are asking for it for years now https://github.com/orgs/community/discussions/41348

**Comment by AYUSHSURYAVANSHI** [comment]
Response #140934
